localstack/services/dynamodb/provider.py

```diff
@@ -549,60 +551,63 @@ def describe_table(self, context: RequestContext, table_name: TableName) -> Desc
+            # DynamoDBLocal refuses to update certain table params and raises.
+            # But we still need to update this info in LocalStack stores
+            if not (exc.code == "ValidationException" and exc.message == "Nothing to update"):
                 raise
 
-            table_name = update_table_input.get("TableName")
-
-            if update_table_input.get("TableClass"):
+            if table_class := update_table_input.get("TableClass"):
                 table_definitions = get_store(
                     context.account_id, context.region
                 ).table_definitions.setdefault(table_name, {})
-                table_definitions["TableClass"] = update_table_input.get("TableClass")
+                table_definitions["TableClass"] = table_class
 
-            if update_table_input.get("ReplicaUpdates"):
-                # update local table props (replicas)
-                table_properties: Dict = get_store(
-                    context.account_id, context.region
-                ).REPLICA_UPDATES
-                table_properties[table_name] = table_replicas = (
-                    table_properties.get(table_name) or {}
-                )
+            if replica_updates := update_table_input.get("ReplicaUpdates"):
+                store = get_store(context.account_id, global_table_region)
+
+                # Dict with source region to set of replicated regions
+                replicas: dict[str, set(str)] = store.REPLICA_UPDATES.get(table_name, {})
+                replicas.setdefault(global_table_region, set())
+
+                for replica_update in replica_updates:
+                    for key, details in replica_update.items():
+                        # Replicated region
+                        target_region = details.get("RegionName")
+
+                        # Check if replicated region is valid
+                        if target_region not in get_valid_regions_for_service("dynamodb"):
+                            raise ValidationException(f"Region {target_region} is not supported")
 
-                # get dict of replicas for the table => {original_region: {regions}}
-                original_region = context.region
-                table_replicas[original_region] = region_replicas = (
-                    table_replicas.get(original_region) or set()
-                )
-                for repl_update in update_table_input["ReplicaUpdates"]:
-                    for key, details in repl_update.items():
-                        # target region
-                        region = details.get("RegionName")
                         match key:
                             case "Create":
-                                table_replicas[original_region].add(region)
-                            case "Update":
-                                table_replicas[original_region] = set(
-                                    [r for r in region_replicas if r == region]
-                                )
```

Comment:
Update operation is removed because existing implementation was incorrect. [Update ](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateReplicationGroupMemberAction.html) is used to update the attributes (like indices, key IDs, etc.) of an existing replica.